src/server/services/validate-upload.js

```diff
@@ -43,6 +43,13 @@ async function validateEcCode ({ upload, records, trns }) {
   const coverSheet = records.find(doc => doc.type === 'cover').content
   const codeString = coverSheet['Detailed Expenditure Category']
 
+  if (codeString === undefined) {
+    return new ValidationError(
+      'Record is missing EC code',
+      { tab: 'cover', row: 2, col: 4 }
```

Comment:
possibly unrelated, our other errors generally refer to the column by it's letter, as that's how they show up in excel. these early validations on the cover sheet are now the odd ones out; in this case, i think it should be `col: 'D'`